lib/type-helpers.js

```diff
@@ -0,0 +1,13 @@
+'use babel'
+
+let isFunction = value => isType(value, 'function')
+
+let isString = value => isType(value, 'string')
+
+var isType = function (value, typeName) {
```

Comment:
Just make this a "real" function instead? `function isType (value, typeName) { `
I guess the same goes for the functions above, but at least here it avoids re-ordering and means we can drop the `var` specifier.